curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64e337e3096b7c1739d934e6.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 64e337e3096b7c1739d934e6
+title: Step 45
+challengeType: 20
+dashedName: step-45
+---
+
+# --description--
+
+Before your recursive call, add a comment saying `move n - 1 disks from source to auxiliary, so they are out of the way`.
```

Comment:
```suggestion
Before your recursive call, add a comment saying `Move n - 1 disks from source to auxiliary, so they are out of the way`.
```
I would suggest starting comments with an uppercase letter, like in the examples from [PEP 8](https://peps.python.org/pep-0008/#inline-comments).